src/impact-dashboard/MitigationInformation.tsx

```diff
@@ -19,6 +24,14 @@ type RowProps = PlannedRelease & {
   updateRelease: (opts: ReleaseUpdate) => void;
 };
 
+const rateOfSpreadDisplayText = {
```

Comment:
You don't have to do this, but I tend to add a type for these things as follows:

```suggestion
const rateOfSpreadDisplayText: {[key: RateOfSpread]: string} = {
```

This makes it so that if you ever add an enum value to `RateOfSpread`, you'll immediately get a ts error here telling you that you didn't account for the new value.

Also consider capitalizing since it's a constant.